wallets/provider-argentx/src/index.ts

```diff
@@ -4,10 +4,11 @@ import type {
   Connect,
   Subscribe,
   WalletInfo,
-} from '@rango-dev/wallets-shared';
+} from '@rango-dev/wallets-core/legacy';
 import type { BlockchainMeta, SignerFactory } from 'rango-types';
 
-import { Networks, WalletTypes } from '@rango-dev/wallets-shared';
+import { Networks } from '@rango-dev/wallets-core/legacy';
+import { WalletTypes } from '@rango-dev/wallets-shared';
```

Comment:
First I tried to remove `Networks` from `shared` but it affected many places, and in some places `core` wasn't available. I decided to keep `Networks` in shared but it actually is a re-export of `wallets-core` (`wallets/shared/src/rango.ts:19`).

So using `import { Networks } from '@rango-dev/wallets-core/legacy'` or `import { Networks } from '@rango-dev/wallets-shared';` basically identical. To avoid put too much effort on changing this, I gave up on changing them in this PR. This change can be continue when we are migrating legacy providers to Hub.

If you think they shouldn't be changed, I can revert back `import { Networks } from '@rango-dev/wallets-core/legacy';` to use `shared` as before.